demisto_sdk/commands/download/downloader.py

```diff
@@ -130,7 +129,6 @@ def __init__(
         run_format: bool = False,
         system: bool = False,
         item_type: str = "",
-        no_code_formatting: bool = False,
```

Comment:
if the `--no-code-formatting` flag is not used anywhere anymore, please remove it and mention it on the changelog.